restapi/server.go

```diff
@@ -61,9 +64,24 @@ func NewServer(loopbackOnly bool, scontext *server.Context, config *server.Confi
 	}
 	ret.mux = http.NewServeMux()
 
+	var tlsConfig *tls.Config
+
+	if config.ACMECert {
+		logging.Info("Using Let's Encrypt for certificates")
+		m := &autocert.Manager{
+			Cache:      autocert.DirCache(config.ACMESecretDir),
+			Prompt:     autocert.AcceptTOS,
+			HostPolicy: autocert.HostWhitelist(config.ACMEHost),
+		}
+
+		go http.ListenAndServe(":http", m.HTTPHandler(nil))
```

Comment:
I had to look at the autocert documentation to understand what this does.  Probably deserves a comment.

Handy, though 👍

Reply:
It pulls up a simple http server to (I believe) the initial cert bootstrapping. Nice bonus is http->https redirects. This is basically a copy and paste from the examples.

Reply:
It only does the redirect.  It was added only recently (I had to update my dependency to use it in office canary).